src/views/Doctor.vue

```diff
@@ -46,6 +46,17 @@ export default {
   components: {
     Card,
   },
+  methods : {
```

Comment:
The function needs to be called when this page opens.  In vuejs life cycle, there is a created method you can use.  In that method you can call the fetchPatients